### ISSUE TITLE
Lots of warnings after upgrade to 6.2.1 

### ISSUE THREAD

**Issue by smallaxe** [mattermost/mattermost-server#19259]
After upgrade to 6.2.1 (from 6.1.0) I see in logs many warnings with this message:

`
Unable to parse developer flag
`

looks like it could be related to this line

https://github.com/mattermost/mattermost-server/blob/aa0b717af7e1325d178537d16fa4b50d3f21edae/web/handlers.go#L107

and latest changes:

https://github.com/mattermost/mattermost-server/commit/9fde5aa5e279ca53fc23adb1b74a14c0d7b041ea#diff-a1712c3db66cc9ee7b31458a5739227d25fc429627dcd439f91dd7a2503f5cb6

My config for `DeveloperFlags` is empty:

```
...
"EnableTesting": false,
"EnableDeveloper": false,
"DeveloperFlags": "",
"EnableOpenTracing": false,
...
```

**Comment by isacikgoz** [comment]
Thanks for the report @smallaxe It seems like the fix you mentioned is not merged into release branch for `v6.2`, however it's included in `v6.3`. I cherry-picked the commit anyway https://github.com/mattermost/mattermost-server/pull/19260

**Comment by agnivade** [comment]
Closing as it's merged now.
